engine/core/src/main/java/org/datacleaner/VersionComparator.java

```diff
@@ -36,32 +36,37 @@ public int compare(String o1, String o2) {
         String[] o2Split = o2.split("\\.");
 
         for (int i = 0; i < Math.min(o1Split.length, o2Split.length); i++) {
-            int snapshotCounter = 0;
+            int specialEndCounter = 0;
             Integer o1Part;
-            if (o1Split[i].endsWith("-SNAPSHOT")) {
-                snapshotCounter++;
-                o1Part = Integer.parseInt(o1Split[i].substring(0, o1Split[i].lastIndexOf("-SNAPSHOT")));
+            if (o1Split[i].contains("-")) {
+                specialEndCounter++;
+                o1Part = Integer.parseInt(o1Split[i].substring(0, o1Split[i].lastIndexOf("-")));
```

Comment:
It is ok.
Comparator was used only for finding newest version between two values. Better code with new suffixes (alfa, RC1) don't change behavior.